recipes-browser/images/core-image-webkit-dev-ci-tools-tests.bb

```diff
@@ -0,0 +1,122 @@
+SUMMARY = "Image with all the required tools to build WPE (using the script build-webkit) \
```

Comment:
Could I suggest a shorter image name: `s/core-image-webkit-dev-ci-tools-tests/`**`webkit-dev-ci-tools`**`/` ?

Reply:
Sounds fine, I will change it.